group_vars/all/release.yml

```diff
@@ -13,10 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## Using this file
-#
-# This file is laid down only once and is used for overriding variables based on the file name.
-# This file will not be overwritten by upgrades.
+# The release tag to use for the repo and venvs
+# This can't be overriden because OSA is using group_vars.
+#openstack_release: "{{ rpc_release }}"
```

Comment:
This will be modified to:
TODO(evrardjp): Uncomment this when OSA has changed its group vars folder location.